tests/meta/demultiplex/conftest.py

```diff
@@ -83,6 +83,18 @@ def fixture_tmp_flow_cell_run_base_path(
     return tmp_flow_cell_run_path
 
 
+@pytest.fixture(name="tmp_flow_cell_demux_base_path")
+def fixture_tmp_flow_cell_demux_base_path(
+    project_dir: Path, bcl2fastq_flow_cell_full_name: str
+) -> Path:
+    """Flow cell run directory in temporary folder."""
+
+    tmp_flow_cell_run_path: Path = Path(project_dir, "demultiplexing-runs")
+    tmp_flow_cell_run_path.mkdir(exist_ok=True, parents=True)
+
+    return tmp_flow_cell_run_path

```

Comment:
Flow cell run is usually associated with the directory before demultiplexing
```suggestion
    return tmp_flow_cell_demux_path

```